### ISSUE TITLE
Playwright timeout when using Quickstart Code

### ISSUE THREAD

**Issue by NelsonDane** [itsjafer/schwab-api#14]
Hello, and thank you for your work on the schwab-api! I have been using this for a long time without any issues when suddenly it started timing out. My login and secrets work correctly when I use the website (https://itsjafer.com/#/reversesplit), but when I use the same values in the Quickstart code (https://github.com/itsjafer/schwab-api#quickstart) I get the following output:

```{python}
Logging into Schwab
Traceback (most recent call last):
  File "/Users/ndane/Documents/Github/auto-rsa/test.py", line 9, in <module>
    logged_in = api.login(
  File "/usr/local/lib/python3.10/site-packages/schwab_api/authentication.py", line 85, in login
    self.page.wait_for_load_state('networkidle')
  File "/usr/local/lib/python3.10/site-packages/playwright/sync_api/_generated.py", line 7371, in wait_for_load_state
    self._sync(self._impl_obj.wait_for_load_state(state=state, timeout=timeout))
  File "/usr/local/lib/python3.10/site-packages/playwright/_impl/_sync_base.py", line 89, in _sync
    return task.result()
  File "/usr/local/lib/python3.10/site-packages/playwright/_impl/_page.py", line 512, in wait_for_load_state
    return await self._main_frame.wait_for_load_state(**locals_to_params(locals()))
  File "/usr/local/lib/python3.10/site-packages/playwright/_impl/_frame.py", line 231, in wait_for_load_state
    return await self._wait_for_load_state_impl(state, timeout)
  File "/usr/local/lib/python3.10/site-packages/playwright/_impl/_frame.py", line 255, in _wait_for_load_state_impl
    await wait_helper.result()
playwright._impl._api_types.TimeoutError: Timeout 30000.0ms exceeded.
=========================== logs ===========================
"commit" event fired
"load" event fired
============================================================
```
I think it may be related to the issue (https://github.com/itsjafer/schwab-api/issues/13) but would appreciate a solution. Thank you!

**Comment by itsjafer** [comment]
Hi, this should be fixed with the version 0.2.3!

**Comment by NelsonDane** [comment]
Everything works again, thank you so much!
